src/Reader.php

```diff
@@ -17,12 +17,19 @@
  * <http://www.doctrine-project.org>.
  */
 
-namespace Doctrine\Common\Annotations;
+declare(strict_types=1);
+
+namespace Doctrine\Annotations;
+
+use ReflectionClass;
+use ReflectionMethod;
+use ReflectionProperty;
```

Comment:
Why import those root namespace classes? I always prefer to read `\ReflectionClass`. If I see `ReflectionClass`, it makes me think it is some kind of custom reflection layer.